packages/router/router.ts

```diff
@@ -1539,6 +1540,9 @@ export function createRouter(init: RouterInit): Router {
 
     pendingNavigationLoadId = ++incrementingLoadId;
     revalidatingFetchers.forEach((rf) => {
+      if (fetchControllers.has(rf.key)) {
+        abortFetcher(rf.key);
+      }
```

Comment:
When we're about to reload/revalidate a `fetcher.load` call, ensure we abort any in-progress loads